packages/analytics-js-common/src/types/ApplicationState.ts

```diff
@@ -119,10 +120,21 @@ export type SessionState = {
 
 export type SourceConfigState = Signal<Source | undefined>;
 
+export type StorageEntry = {
+  storage: StorageType;
+  key: string;
```

Comment:
What does `key` hold?

Reply:
Storage key: 'rl_user_id'

Reply:
Ok. Then let's rename the fields to `type` and `key`.